### PR TITLE
Resizing Instructions Tab Content

### PR DESCRIPTION
**Pull request by dmcavoy**
Instructions area now resizes to show the full feedback tab contents when selected. All other tabs stay with the same behavior they had before. Once the instructions area resizes for the feedback tab though it keeps that size until manually resized by the user.

## Full Size Feedback Area
Screen Shot 2019-03-12 at 11 21 01 AM

## Demo of resizing
Resize-Instructions-Feedback 2

### CODE REVIEW

File: apps/src/templates/instructions/TopInstructionsCSP.jsx [hunk 1]
```diff
@@ -243,6 +243,10 @@ class TopInstructions extends Component {
       RESIZER_HEIGHT;
 
     this.props.setInstructionsMaxHeightNeeded(maxNeededHeight);
+    // Force instruction area to resize to show full feedback area
+    if (this.state.tabSelected === TabType.COMMENTS) {
+      this.props.setInstructionsRenderedHeight(maxNeededHeight);
+    }
```
Comment (davidsbailey) [inline apps/src/templates/instructions/TopInstructionsCSP.jsx:249]: why is this needed only for the comments/feedback area?

Comment (dmcavoy) [reply apps/src/templates/instructions/TopInstructionsCSP.jsx:249]: this forces the area to resize to the full max needed height for the content. If you do this for something like instructions it can take over the whole screen which I'm guessing we did not want since that's not the way its been in the past.

**Review by davidsbailey — APPROVED** [review]
